utilization.md

```diff
@@ -161,48 +215,34 @@ Kokkos::View<double**[25]> view3D("view3D", 50, 42, 25);
-| `size()`    | Returns the total number of elements in the view                                                                      |
-| `rank()`    | Returns the number of dimensions                                                                                      |
-| `extent()`  | Returns the number of elements in each dimension                                                                      |
-| `layout()`  | Returns the layout of the view                                                                                        |
-| `resize()`  | Reallocates a view to have the new dimensions; can grow or shrink, and will preserve content of the common subextents |
-| `realloc()` | Reallocates a view to have the new dimensions; can grow or shrink, and will not preserve content                      |
-| `data()`    | Returns a pointer to the underlying data                                                                              |
-
-<!--#ifndef PRINT-->
 <img title="Doc" alt="Doc" src="./images/doc_txt.svg" height="25"> https://kokkos.org/kokkos-core-wiki/API/core/view/view.html#data-layout-dimensions-strides
 <!--#endif-->
 
-### Memory Layout
+### Memory Layouts
 
-Layout Determines the mapping of indices into the underlying 1D memory storage:
+| Layout                 | Description                                                                                                 | For |
+|------------------------|-------------------------------------------------------------------------------------------------------------|-----|
+| `Kokkos::LayoutRight`  | Strides increase from the right most to the left most dimension, also known as row-major or C-like          | CPU |
+| `Kokkos::LayoutLeft`   | Strides increase from the left most to the right most dimension, also known as column-major or Fortran-like | GPU |
+| `Kokkos::LayoutStride` | Strides can be arbitrary for each dimension                                                                 |     |
 
-| Layout                   | Description                                                     | Other name                   | Efficient for |
-|--------------------------|-----------------------------------------------------------------|------------------------------|---------------|
-| `Kokkos::LayoutRight()`  | Strides increase from the right most to the left most dimension | row-major or C-like          | CPU           |
-| `Kokkos::LayoutLeft()`   | Strides increase from the left most to the right most dimension | column-major or Fortran-like | GPU           |
-| `Kokkos::LayoutStride()` | Strides can arbitrary for each dimension                        |                              |               |
-
-If no layouts are specified, the most efficient one for the memory space is used.
-For performance, memory access patterns must result in caching on a CPU and coalescing on a GPU.
+By default, the most efficient layout for the current memory space is used.
```

Comment:
A little bit strong. The default is a reasonable choice for loops on the high frequency index.